src/renderer/shared/wallet/Wallet.js

```diff
@@ -6,15 +6,15 @@ import { DEFAULT_LANGUAGE } from 'shared/values/languages';
 import simpleDecrypt from 'shared/util/simpleDecrypt';
 import { NEO, ETH } from 'shared/values/coins';
 
-const Wallet = ({ encryptedMnemonic, passphrase, wallet }) => {
+const Wallet = async ({ encryptedMnemonic, passphrase, wallet }) => {
   const mnemonic = attempt(simpleDecrypt, encryptedMnemonic, passphrase);
 
   // Validate mnemnoic
   if (isError(mnemonic) || !bip39.validateMnemonic(mnemonic, bip39.wordlists[DEFAULT_LANGUAGE])) {
     throw new Error("Please make sure you've entered the correct password.");
   }
 
-  const seed = bip39.mnemonicToSeed(mnemonic, passphrase);
+  const seed = await bip39.mnemonicToSeed(mnemonic, passphrase);
```

Comment:
aaaaaaaaaaaah this changed to an async function since the 3.0.x release. 
There's also a sync version if I am not mistaken, but this is fine aswell :)

Reply:
Better safe than sorry? 😎